x-pack/plugins/uptime/server/lib/requests/get_monitor_states.ts

```diff
@@ -152,7 +152,7 @@ export const getHistogramForMonitors = async (
   };
   const { body: result } = await queryContext.search({ body: params });
 
-  const histoBuckets: any[] = result.aggregations?.histogram.buckets ?? [];
+  const histoBuckets: any[] = (result.aggregations as any).histogram.buckets ?? [];
```

Comment:
i think optional chaining should remain here.

Reply:
Do you mean in runtime? I will put it back